packages/tooling/fast-tooling/src/data-utilities/shortcuts.spec.md

```diff
@@ -0,0 +1,126 @@
+# Abstract
+
+The shortcuts utility accepts an instance of the `MessageSystem` and through this sends information about available shortcuts and events.
+
+## Instantiating
+
+All the `Shortcuts` class needs to instantiate is an instance of the `MessageSystem`. It should then fire a custom event notifying all other items registered to the message system that shortcuts are available, with a listener that will capture keypress and identify which shortcut action to execute.
+
+```typescript
+import { Shortcut } from "@microsoft/fast-tooling"
+
+new Shortcuts({
+    messageSystem: fastMessageSystem
+});
+```
+
+## Adding shortcut actions
+
+Shortcut actions can either be custom or imported from a set of actions available.
+
+```typescript
+import { DuplicateShortcutAction, ShortcutAction, Shortcuts } from "@microsoft/fast-tooling"
+import { keyCodeAlt, keyCodeAltName , keyCodeTab, keyCodeTabName } from "@microsoft/fast-web-utilities"
+
+new Shortcuts({
+    messageSystem: fastMessageSystem,
+    actions: [
+        // Custom shortcut
+        new ShortcutAction({
+            name: "Random action",
+            keyCodes: [
+                {
```

Comment:
Is the expectation here that the user is pressing `alt` and `tab`?  I wonder if it would make sense to map this more closely to the platform event object itself, where `altKey`, `ctrolKey`, and `shiftKey` are properties on the event object:

```js
new shortcutAction({
  keyCodes: [{name: keyCodeTabName, value: keyCodeTab, altKey: true}]
})
```

Reply:
In hindsight, that was a bad example, that's a good point I'll update.